docs: guide for token detection (#36)
rotki/docs: usage-guides/assets.md

![Whitelist asset](/images/asset_whitelist_menu.png)

## Whitelisting and Re-detecting Missing Tokens

If a token doesn't appear in your balances:

### Check Asset Status

> Go to the Manage Assets -> Assets page
> Use the search to find the token by name or address
> Verify if the token is ignored
> If ignored, click to whitelist it

Reviewer: ```suggestion
Go to the Manage Assets -> Assets page. Use the search to find the token by name or address. Verify if the token is ignored. If ignored, unignore it. If you did not ignore it on purpose click to whitelist it so the spam detection algorithm does not falsely flag it as spam.
```

**All Accounts Per-Chain Detection**

> Go to Account & Balances -> Blockchain Balances. Scroll down to EVM accounts and click on Re-detect tokens (🔃). Here you can select specific chains to detect tokens for all your wallets.

Reviewer: ```suggestion
Go to Account & Balances -> Blockchain Balances. Scroll down to EVM accounts and click on Re-detect tokens (🔃). Here you can select specific chains to detect tokens for all your wallets. This will take a bit of time depending on tracked addresses per chain. The slowest chain for detection is ethereum.
```

### Re-detect After Whitelisting

> Go to Account & Balances -> Blockchain Balances
> Scroll down to EVM accounts and click on Re-detect tokens (🔃) Here you can specify chains for re-detection.
> ![Example of re-detecting tokens for all EVM chains in the Blockchain Balances section.](/images/redetect-tokens-chain.png)
> 
> If still not appearing, try the full token detection from the dashboard by clicking on 🔃 toggle.
> ![Example of performing a full token detection using the re-detect toggle on the dashboard.](/images/redetect-tokens-dashboard.png)

Reviewer: Here you should specify the different types of detection. The best way to detect tokens for a single address is go to blockchain & accounts balances, go to the address, go the the per chain view and click detect tokens there.

```suggestion
Go to Account & Balances -> Blockchain Balances. Scroll down to EVM accounts and click on Re-detect tokens (🔃) Here you can specify chains for re-detection. ![Example of re-detecting tokens for all EVM chains in the Blockchain Balances section.](/images/redetect-tokens-chain.png)

If still not appearing, try the full token detection from the dashboard by clicking on 🔃 toggle. ![Example of performing a full token detection using the re-detect toggle on the dashboard.](/images/redetect-tokens-dashboard.png)
```

> 

Reviewer: In the last commit you seem to have completely removed the part you added in the first commit which was what I asked you to make this PR for. Explaining the common problem that people have with accidentally ignoring assets or having it flagged by the spam detection method.

Reviewer 2: I just checked, for this part, it has been added here https://docs.rotki.com/usage-guides/troubleshooting.html#my-asset-is-not-showing-in-rotki

Reviewer 2: @cleanerzkp Can you check if the information is duplicated with this one https://docs.rotki.com/usage-guides/troubleshooting.html#my-asset-is-not-showing-in-rotki , or perhaps if you need to put the link to there

> For a complete re-detection of all tokens across all your wallets and all supported chains, use the re-detect toggle (🔃) in the main dashboard view. This will perform a full scan of your wallets on every chain.

Reviewer: Do not conflate the world wallet with an address. Using "wallet" for an address is a mistake and misnomer used by people who don't know the difference.

```suggestion
For a complete re-detection of all tokens across all your addresses and all supported chains, use the re-detect toggle (🔃) in the main dashboard view. This will perform a full scan of your addresses on every chain but will probably take quite a long time depending on amount of tracked addresses and chains.
```

![Whitelist asset](/images/asset_whitelist_menu.png)

## Whitelisting and Re-detecting Missing Tokens

> If a token doesn't appear in your balances, there are several ways to detect tokens:

Reviewer: ```suggestion
If a token doesn't appear in your balances, there are several ways to troubleshoot this. First you should check if it's been automatically or manually ignored. And then run a manual token detection.
```

> ### Check for Spam Detection

Reviewer: ```suggestion
### Check Token is not Ignored or Marked as Spam
```

Go to the Manage Assets -> Assets page. Use the search to find the token by name or address. Verify if the token is ignored. If ignored, unignore it. If you did not ignore it on purpose click to whitelist it so the spam detection algorithm does not falsely flag it as spam.

![Example of managing assets and whitelisting a token in the Manage Assets page.](/images/manage-assets-refresh.png)

### Token Detection Methods

**Single Account Per-Chain Detection**

> The best way to detect tokens for a single address is to go to blockchain & accounts balances, scroll down to the EVM accounts vuew, expand the dropdown at the account, go to the per chain tab and click on the detect tokens toggle 🔃.

Reviewer: ```suggestion
The best way to detect tokens for a single address is to go to blockchain & accounts balances, scroll down to the EVM accounts view, expand the dropdown at the account, go to the per chain tab and click on the detect tokens toggle 🔃.
```
